README.md

```diff
@@ -1,21 +1,38 @@
 
-If you have any feedback or are interested in what we are doing please [contact us](https://ably.com/contact).
+If you have any feedback or are interested in what we are doing please reach out to us at [beta@ably.com](mailto:beta@ably.com).
 
-API for managing and accessing Ably Spaces.
+## Concepts
+The Ably Spaces library provides purpose-built methods and properties to enable multiplayer collaboration in any web application. You can set up a collaborative "space" and subscribe to different kind of updates to see who else is in the space with you, their live location within the app and any updates they are making.
 
-**Class definitions are available [here](/docs/class-definitions.md).**
+This library is built as an extention to our [existing JS SDK](https://github.com/ably/ably-js), so you'll need to an Ably JS client to be able to use this library. The usage instructions will help you set that up.
 
-### Usage
+### Glossary
+As part of this library, we are introducing some new concepts:
 
-To use the Space API, you'll need to have an Ably client already set up and authenticated.
-To get started quickly, you can use [basic authentication](https://ably.com/docs/realtime/authentication#basic-authentication)
-by creating an API key with the **publish**, **subscribe** and **presence** capabilities enabled.
+1. Space: Avirtual collaborative space where members can collaborate in realtime with each other.
+2. Members: Users connected to the virtual collaborative space. The same user (identified by the clientId) can be present multiple times via different connections.
+3. Avatar Stack: A visual representation of the currently present and recently left list of members in the collaborative space. This is usually presented as a stack of profile pictures or initials along with other profile information such as name or role.
+3. (Coming soon) User Location: The live location of a member within the app. Depending on the nature of the app, this could be a slide number, a URL, a cell id, component id, etc.
+4. (Coming soon) Live cursor: The pointer location of a member within the app. 
 
 
-Here's an example of how you can create a new instance of the client and pass it to the Spaces constructor:
+## Usage
+
+**Class definitions are available [in the docs folder](/docs/class-definitions.md).**
+
+### Pre-requisites
+
+To use the Space API, you'll need to have an Ably JS client set up and authenticated. To get started quickly, you can use [basic authentication](https://ably.com/docs/realtime/authentication#basic-authentication) which only requires an Ably API Key. 
+
+1. [Sign up to a free Ably account](https://ably.com/signup)
+2. [Get your API Key](https://faqs.ably.com/setting-up-and-managing-api-keys) with publish, subscribe and presence capabilities enabled.
+
+### Quickstart 
+
+Create a new instance of the promisified realtime client and pass that as a parameter to the Spaces constructor:
```

Comment:
Technically `ably/promise` is promisified, but in my opinion it would be better if we referred to it simply as the promise version of the realtime client (or just `ably/promise`)